commands/certmanager/certificates.go

```diff
@@ -0,0 +1,98 @@
+package certmanager
+
+import (
+	"github.com/fatih/structs"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/ionos-cloud/ionosctl/pkg/constants"
+	"github.com/ionos-cloud/ionosctl/pkg/core"
+	"github.com/ionos-cloud/ionosctl/pkg/printer"
+	ionoscloud "github.com/ionos-cloud/sdk-go-cert-manager"
+)
+
+func CertCmd() *core.Command {
+	certCmd := &core.Command{
+		Command: &cobra.Command{
+			Use:              "certificate-manager",
+			Short:            "Certificate Manager Operations",
+			Long:             "The sub-commands of `ionosctl certificate-manager` allows you to manage the SSL Certificates under your account.",
+			TraverseChildren: true,
+		},
+	}
+	certCmd.AddCommand(CertGetCmd())
+	certCmd.AddCommand(CertCreateCmd())
+	certCmd.AddCommand(CertListCmd())
+	certCmd.AddCommand(CertDeleteCmd())
+	certCmd.AddCommand(CertGetApiVersionCmd())
+	certCmd.AddCommand(CertUpdateCmd())
+	return certCmd
+}
+
+func getCertPrint(resp *ionoscloud.APIResponse, c *core.CommandConfig, cert *[]ionoscloud.CertificateDto) printer.Result {
+	r := printer.Result{}
+	if c != nil {
+		if resp != nil {
+			r.Resource = c.Resource
+			r.Verb = c.Verb
+			r.WaitForState = viper.GetBool(core.GetFlagName(c.NS, constants.ArgWaitForRequest)) // this boolean is duplicated everywhere just to do an append of `& wait` to a verbose message
+		}
+		if cert != nil {
+			r.OutputJSON = cert
+			r.KeyValue = getCertRows(cert)                                                              // map header -> rows
+			r.Columns = getCertHeaders(viper.GetStringSlice(core.GetFlagName(c.NS, constants.ArgCols))) // headers
+		}
+	}
+	return r
+}
+
+func printProperties(value *ionoscloud.CertificateDto, c *core.CommandConfig, flags []bool) string {
+	var printString string
+	if flags[0] {
+		printString += *value.Properties.Certificate
+	}
+	if flags[1] {
+		printString += *value.Properties.CertificateChain
+	}
+	return printString
+}
```

Comment:
- No bool slice - It allows passing an infinite number of bools. Must restrict to "either this or that flag"

- Dont concatenate into printString. If I pass printProperties(true, true) it would return both cert and cert chain.

- Use the SDK getters ` *value.GetProperties().getCertificateChain()`

- Param `c` not used - remove

- Signature should match behaivour:
    -  Func doesn't print anything: prefix `get` instead. 
    - require only one bool param instead of a slice of unknown length. If this bool set, print Cert. If not, print CertChain

```
func getOnlyCertOrCertChain(value *ionoscloud.CertificateDto, getCertElseChain bool) string {
	if getCertElseChain {
		return *value.getProperties().getCertificate()
	}
	return *value.getProperties().getCertificateChain()
}
```